spec/requests/api/generic_events_controller_create_spec.rb

```diff
@@ -47,8 +42,42 @@
       before { do_post }
     end
 
+    context 'when event-specific relationships are in the details attribute' do
```

Comment:
spoke in person: We shouldn't allow accidentally setting relationships through the details attribute, we agreed that this test will be marked as pending, and will be tackled in future PRs to remove the creation of location through details, and only allow it through relationships